packages/fxa-auth-server/types/config/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// This interface should be expanded on
+// as we slowly move over to TypeScript
```

Comment:
It actually might be easier just change the config file to `.ts`, install `@types/convict`, and export `typeof conf` in the config ts.

Reply:
So many implications for changing the config file to typescript 😆